programs/ssl/ssl_base64_dump.c

```diff
@@ -177,6 +182,14 @@ void print_hex( const uint8_t *b, size_t len )
     fflush(stdout);
 }
 
+void print_time( const time_t *time )
+{
+    char buf[20];
```

Comment:
Is 20 always enough?

Reply:
Yes, the output format has a constant length "2020-01-23 13:05:59"